lib/ansible/playbook/role/__init__.py

```diff
@@ -274,22 +275,49 @@ def _load_role_data(self, role_include, parent_role=None):
                 raise AnsibleParserError("The handlers/main.yml file for role '%s' must contain a list of tasks" % self._role_name,
                                          obj=handler_data, orig_exc=e)
 
-    def _prepend_validation_task(self, task_data):
+    def _get_role_argspecs(self):
+        '''Get the role argument spec data.
```

Comment:
PEP 257 suggest triple double quotes for docstrings.
```suggestion
        """Get the role argument spec data.
```